### PR TITLE
Added new Spring boot admin guide

### PR DESCRIPTION
**Pull request by benwilcock**
Added new Spring boot admin guide

### CODE REVIEW

File: content/guides/inspect-apps-with-spring-boot-admin/index.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+
+As this is a brand new project, Git source code tracking must be added. First, initialize the project with `git init`, then add all the source code to version control with `git add`, and finally, create the first commit with `git commit`. Here are the commands you need:
+
+```bash
+git init # Initialize the project as a Git project
+git add --all # Add all the source code
+git commit -am "initial commit" # Create the first commit
+```
+
+The code must then be pushed to a new remote repository on GitHub. You can do this using the GitHub command line tool as follows.
```
Comment (BrianMMcClain) [inline content/guides/inspect-apps-with-spring-boot-admin/index.md:173]:
```suggestion
The code must then be pushed to a new remote repository on GitHub. You can do this using the [GitHub command line tool](https://cli.github.com/) as follows. 
```

File: content/guides/inspect-apps-with-spring-boot-admin/index.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+```
+
+![Image of the tanzu cli showing the Spring Boot Admin application's URL](images/SbaUrl.png "Image of the tanzu cli showing the Spring Boot Admin application's URL")
+
+Open the URL in your browser.
+
+![Image of Spring Boot Admin running on TAP](images/SbaOnTapClip.gif "Image showing a Spring Boot Admin running on TAP.")
+
+{{%warning%}}
+If you're not using Tanzu Application Platform, expose the application as a Kubernetes service in the usual way and then navigate your browser to the application's URL.
```
Comment (BrianMMcClain) [inline content/guides/inspect-apps-with-spring-boot-admin/index.md:227]: Do we want to provide a link to a resource that describes how to do this for reference? Even just a link to the [kubectl expose](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#expose) docs. We also have a piece on [Service Routing](https://tanzu.vmware.com/developer/guides/service-routing/) but perhaps that's a but overwhelming? What do you think?

**Review by BrianMMcClain — CHANGES_REQUESTED** [review]
Awesome stuff @benwilcock! Just had a couple pieces of feedback to add a couple of reference links, wasn't sure what would be best on the second one but I'm open to suggestions :)

File: content/guides/inspect-apps-with-spring-boot-admin/index.md (hunk 2 above)
Comment (benwilcock) [reply content/guides/inspect-apps-with-spring-boot-admin/index.md:227]: Maybe go with a link to the first one - kubectl expose. That works for me. Better simple docs than a whole guide I think.

File: content/guides/inspect-apps-with-spring-boot-admin/index.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
```
Comment (BrianMMcClain) [inline content/guides/inspect-apps-with-spring-boot-admin/index.md:227]:
```suggestion
If you're not using Tanzu Application Platform, [expose the application as a Kubernetes service](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#expose) in the usual way and then navigate your browser to the application's URL.
```

**Review by BrianMMcClain — APPROVED** [review]

**Review by BrianMMcClain — APPROVED** [review]

**Pull request merged**